Demo/Demo/ViewModels/BaseViewModel.swift

```diff
@@ -100,9 +100,8 @@ class BaseViewModel: ObservableObject, PayPalDelegate {
         let cardRequest = CardRequest(orderID: orderID, card: card)
 
         do {
-            let result = try await cardClient.approveOrder(request: cardRequest)
+            _ = try await cardClient.approveOrder(request: cardRequest)
             updateTitle("\(DemoSettings.intent.rawValue.capitalized) status: APPROVED")
-            await processOrder(orderID: result.orderID)
```

Comment:
This fails every time with an HTTP 500. I think there's a problem with the sample merchant server. Ended up removing since we don't make this call on Android, and the orders v2 `approve-order` request is working correctly.